learning/courses/quantum-diagonalization-algorithms/exam.mdx

```diff
@@ -7,6 +7,10 @@ description: Take a 20-question exam on the IBM Training site to test your quant
 
 ![A square badge stating the name of this course, foundational level, and issued by IBM Quantum&reg;.](/learning/images/courses/quantum-diagonalization-algorithms/badge.avif)
 
+## Post-Course Survey
+
+Congratulations on completing this course! Please take a moment to help us improve our course by filling out the following [quick survey](https://your.feedback.ibm.com/jfe/form/SV_6S5Y3mnEAqGmM0m). Your feedback will be used to enhance 
```

Comment:
```suggestion
Congratulations on completing this course! Please take a moment to help us improve our course by filling out the following [quick survey](https://your.feedback.ibm.com/jfe/form/SV_6S5Y3mnEAqGmM0m). Your feedback will be used to enhance our content offering and user experience. Thank you! 
```

Reply:
Added back in https://github.com/Qiskit/documentation/pull/3476/commits/b13315f46a32270866baa1a7b868b8f651f293e4